spec/posthog/feature_flag_spec.rb

```diff
@@ -0,0 +1,2951 @@
+require 'spec_helper'
+
+
+class PostHog
+
+  RSpec::Support::ObjectFormatter.default_instance.max_formatted_output_length = nil
+
+  describe 'local evaluation' do
+  
+    it 'evaluates person properties' do
+      api_feature_flag_res = {
+        "flags": [
+          {
+            "id": 1,
+            "name": "Beta Feature",
+            "key": "person-flag",
+            "is_simple_flag": true,
+            "active": true,
+            "filters": {
+                "groups": [
+                    {
+                        "properties": [
+                            {
+                                "key": "region",
+                                "operator": "exact",
+                                "value": ["USA"],
+                                "type": "person",
+                            }
+                        ],
+                        "rollout_percentage": 100,
+                    }
+                ],
+            },
+          },]
+      }
+      stub_request(
+        :get,
+        'https://app.posthog.com/api/feature_flag/local_evaluation?token=testsecret'
+      ).to_return(status: 200, body: api_feature_flag_res.to_json)
+
+      # shouldn't call decide
+      stub_request(:post, 'https://app.posthog.com/decide/?v=2')
```

Comment:
Note that this isn't ideal - any other request with a slightly different url would still go live.